doc/cygnus-common/installation_and_administration_guide/running_as_service.md

```diff
@@ -0,0 +1,24 @@
+#Running Cygnus as a service
```

Comment:
Cygnus -> cygnus-common

This must be checked in all the entries about running